internal/js-ast-utils/isBinary.test.ts

```diff
@@ -0,0 +1,34 @@
+import {test} from "rome";
+import {isBinary} from "./isBinary";
+import {parseJS} from "@internal/js-parser";
+import {jsExpressionStatement} from "@internal/ast";
+
+function binaryExpressionHelper(input: string) {
```

Comment:
```suggestion
function binaryExpressionHelper(input: string): boolean {
```

Nit: Explicit return types where possible.